pubspec.yaml

```diff
@@ -24,8 +24,8 @@ environment:
   # that by the time we want to release, these will have become stable.
   # TODO: Before general release, switch to stable Flutter and Dart versions,
   #   or pin exact versions: https://github.com/zulip/zulip-flutter/issues/15
-  sdk: '>=3.4.0-71.0.dev <4.0.0'
-  flutter: '>=3.19.0-9.0.pre.169'
+  sdk: '>=3.2.6 <4.0.0'
+  flutter: '>=3.16.9'
```

Comment:
Don't downgrade our Flutter version :slightly_smiling_face: 

If this change works for you, it's OK to have locally; but I'd encourage you to use the Flutter main channel instead as described in our README file. In any case it's not a change we'd want to merge.

Reply:
I appreciate your feedback and concerns. I'll stick to the Flutter main channel as per the README file.